controller/proxy-injector/webhook.go

```diff
@@ -89,6 +89,11 @@ func (w *Webhook) decode(data []byte) (*admissionv1beta1.AdmissionReview, error)
 func (w *Webhook) inject(request *admissionv1beta1.AdmissionRequest) (*admissionv1beta1.AdmissionResponse, error) {
 	log.Debugf("request object bytes: %s", request.Object.Raw)
 
+	admissionResponse := &admissionv1beta1.AdmissionResponse{
```

Comment:
mind moving this down to directly above the first time it's returned? i'm extra paranoid about myself or other devs accidentally returning this object with `Allowed: true` when we shouldn't.